src/ttltrack.rs

```diff
@@ -0,0 +1,422 @@
+/*
+参考ttltrack 注释全部由copilot生成
+ */
+use std::collections::HashMap;
+use std::time::{Duration, SystemTime};
+
+// 常量定义
+const TCP_CONNRECORD_KEY_LEN: usize = 37; // TCP连接记录键的长度
+const TCP_CLEANUP_INTERVAL_SEC: u64 = 30; // 清理间隔时间（秒）
+
+// TCP连接跟踪信息结构体
+#[derive(Clone)]
+pub struct TcpConntrackInfo {
+    pub is_ipv6: u8, // 是否为IPv6
+    pub ttl: u8, // 生存时间
+    pub srcip: [u32; 4], // 源IP地址
+    pub srcport: u16, // 源端口
```

Comment:
Please use semantic styled naming conventions like `src_port` and `src_ip`, I'll setup cspell and it's workflow later.